js/src/views/Accounts/List/list.js

```diff
@@ -31,7 +34,11 @@ export default class List extends Component {
     empty: PropTypes.bool,
     order: PropTypes.string,
     orderFallback: PropTypes.string,
-    handleAddSearchToken: PropTypes.func
+    certifications: PropTypes.object.isRequired,
```

Comment:
I don't have a system of ordering "regular" props, but I have at least been trying to put actions at the end and props passed from the parent at the beginning.